R/lancelot.R

```diff
@@ -2163,6 +2236,10 @@ lancelot_prepare_data <- function(data) {
     }
   } else {
     lapply(split(data, data$population), function(x) {
+      if (check_deaths(x) > 1) {
+        stop(sprintf("cannot fit to all ages aggregated for deaths if fitting
```

Comment:
Can you please add a test for this condition

Reply:
Thanks Rich, will do